.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Given that we're sensitive to the version of indent, I'd rather pin a specific version of Ubuntu.

Reply:
Indeed, `ubuntu-latest` still resolves to 18.04, and 20.04 doesn't work! There's a problem for the future ;)